packages/synapse-react-client/test/lib/utils/functions/sqlFunctions.test.ts

```diff
@@ -217,4 +255,17 @@ describe('generateQueryFilterFromSearchParams', () => {
     const result = generateQueryFilterFromSearchParams(searchParams, operator)
     expect(result).toStrictEqual(expectedResults)
   })
+
+  it('handles every filter type', () => {
```

Comment:
If we add a new filter method, this test will fail unless we add a new case to `generateQueryFilterFromSearchParams`